src/149 - CDMyRulesEngine/CDMyRulesEngine/CDMyRulesEngine.csproj

```diff
@@ -0,0 +1,67 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" >
+  <!--
+ SPDX-FileCopyrightText: 2009-2020 TRUMPF Laser GmbH, authors: C-Labs
+ SPDX-License-Identifier: MPL-2.0
+ -->
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net45;net40;net35</TargetFrameworks>
```

Comment:
Should we remove net40 target then?

Reply:
if we remove 40 then also remove 3.5 either both or none

Reply:
If it doesn't work why compile for it? (referring to the comment below, BTW).

Reply:
The comment was outdated